java/src/test/java/io/cucumber/gherkin/GherkinParserTest.java

```diff
@@ -42,10 +45,43 @@ void use_this_in_readme() {
     @Test
     void test_for_profiler_parser() throws IOException {
         for (int i = 0; i < 1000; i++) {
-            GherkinParser.builder().build().parse(Paths.get("../testdata/good/very_long.feature"));
+            GherkinParser.builder()
```

Comment:
Kinda surprised this isn't extracted out of the loop. I would not expect that reusing the parser would influence the profiler.